src/NuGet/Microsoft.Net.Compilers.Toolset/Framework/Microsoft.Net.Compilers.Framework.Package.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+
+    <IsPackable>true</IsPackable>
+    <NuspecPackageId>Microsoft.Net.Compilers.Framework</NuspecPackageId>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <DevelopmentDependency>true</DevelopmentDependency>
+    <PackageDescription>
+      .NET Framework Compilers Toolset Package
+
+      Referencing this package will cause the project to be built using the C# and Visual Basic framework compilers contained in the package, as opposed to the version installed with MSBuild.
+
+      This package is an implementation detail of a .NET SDK feature. Directly referencing this package is not supported. 
```

Comment:
Are we planning on pushing this package to nuget.org?

Reply:
Yes. The hope is that we can do this as unlisted.

Reply:
We might need to update the nuget publish tool to do that automatically then.

Reply:
Where is this tool so I can update it?

Reply:
https://github.com/dotnet/roslyn-tools/blob/main/src/dotnet-roslyn-tools/NuGet/NuGetPublish.cs

Reply:
Okay will update that once we merge.